reascripts/ReaSpeech/source/TranscriptUI.lua

```diff
@@ -57,6 +57,14 @@ function TranscriptUI:init()
   self:init_layouts()
 end
 
+function TranscriptUI:_clipper()
+  if not ImGui.ValidatePtr(self.clipper, 'ImGui_ListClipper*') then
+    self.clipper = ImGui.CreateListClipper(ctx)
```

Comment:
I would typically swap the spelling of the method vs. instance variable name here: `TranscriptUI:clipper()` and `self._clipper`, since there's no good reason to use `self.clipper` directly. Or, to keep a notion of privacy for the method, `TranscriptUI:_get_clipper()`